ChangeLog.d/driver-only-hashes.txt

```diff
@@ -8,12 +8,16 @@ Features
      are only provided by PSA drivers. In these configurations, you need to
      call `psa_crypto_init()` before you call any function from those
      modules; this is not required in configurations where the built-in
-     implementation is still available. Note that some crypto modules and
-     features still depend on the built-in implementation of hashes:
-     MBEDTLS_HKDF_C (but the PSA HKDF function do not depend on it),
-     MBEDTLS_ENTROPY_C, MBEDTLS_HMAC_DRBG_C and MBEDTLS_ECDSA_DETERMINISTIC.
-     In particular, for now, compiling without built-in hashes requires use
-     of MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG.
+     implementation is still available. Note that for modules that use MD
+     (RSA, PKCS5, PKCS12, EC J-PAKE) in builds that have MBEDTLS_MD_C enabled,
+     all hashes used with those modules need to be built-in, as drivers are only
+     used when MBEDTLS_MD_C is disabled; configurations where some hashes are
+     available as built-ins, and some only from drivers, are currently not
+     supported. Also note that some crypto modules and features still depend on
+     the built-in implementation of hashes: MBEDTLS_HKDF_C (but the PSA HKDF
+     functions do not depend on it), MBEDTLS_ENTROPY_C, MBEDTLS_HMAC_DRBG_C and
+     MBEDTLS_ECDSA_DETERMINISTIC. In particular, for now, compiling without
+     built-in hashes requires use of MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG.
```

Comment:
Except for builds without entropy (e.g. only doing hashes, MAc, symmetric decryption, and signature verification).

Reply:
Ah, good point, you can't yet have PSA hashes without having a RNG.